frontend/www/js/omegaup/components/user/ViewProfile.test.ts

```diff
@@ -59,4 +148,86 @@ describe('Profilev2.vue', () => {
     });
     expect(wrapper.find('[data-profile-navtabs]').exists()).toBe(true);
   });
+
+  it('Should display all contests', async () => {
+    const badge_alias = 'contestManager';
+    const wrapper = mount(user_ViewProfile, {
+      propsData: {
+        profile,
+        data,
+        profileBadges: new Set(badge_alias) as Set<string>,
```

Comment:
hmm creo que esto estaba mal desde antes

```suggestion
        profileBadges: new Set([badgeAlias]),
```

en particular:

* el constructor de [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set) recibe un iterable con los miembros.
* no deberíamos de usar `as` si se puede evitar.